guides/Tong/README.md

```diff
@@ -0,0 +1,173 @@
+## Written by Sharon Tong
+_________________________
+
+### Contact
+- Slack: @Sharon on wcscusf.slack.com
+- [LinkedIn](linkedin.com/in/sharon-tong)
+- Email: sharont1@mail.usf.edu
+### Table of Contents
+1. [Prerequisites](#prereq)
+2. [Summary](#summary)
+3. [Installing PFsense](#installation)
+4. [Configuring PFsense](#configuration)
+5. [Troubleshooting](#troubleshooting)
+
+## Prerequisites <a id="prereq"></a>
+1. Have your [virtual environment](https://www.virtualbox.org/wiki/Downloads) configured.
+2. Have [WinRAR](https://www.win-rar.com/start.html?&L=0) installed on your computer.
+3. Have your ISP gateway running (we'll be using [Ubuntu ISP](https://silexone.github.io/guides/nestor/ISPsetup.html) here).
+
+## Summary <a id="summary"></a>
+PFsense is an open source firewall that is based on the FreeBSD operating system. Here, we will learn how to install and setup PFsense in a virtualization software, in this case, VirtualBox.
```

Comment:
Now that you fully setup pfsense you should have a greater understanding. Try to further summarize your guide. Maybe images also.